guides/common/assembly_installing-capsule-server.adoc

```diff
@@ -10,18 +10,15 @@ For more information, see {InstallingSmartProxyDocURL}preparing-environment-for-
 ifdef::satellite,katello[]
 // Registering {SmartProxyServer} to {ProjectServer}
 include::modules/proc_registering-capsule-to-satellite-server.adoc[leveloffset=+1]
+endif::[]
```

Comment:
Another missed endif? I think we need a checker for that.

Reply:
No, the endif was in the old version on line 22. However, configuring repositories is not limited to satellite and katello so the ifdef here should be closed earlier. Technically it could have been on line 18 too, but this keeps it closer to the opening ifdef.